robotbase/src/main/java/com/team2357/frc2022/controls/IntakeDriveControls.java

```diff
@@ -99,8 +99,7 @@ public IntakeDriveControls build() {
             // Intake Mode Bindings
             if (m_intakeSub != null) {
                     m_IntakeDriverControls.m_leftTrigger.whileActiveOnce(
-                            new IntakeRollerCommand(m_intakeSub,
-                                    m_IntakeDriverControls.getControllerAxisValue(Axis.kLeftTrigger)),false);
+                            new IntakeSequenceCommandGroup(m_intakeSub));
```

Comment:
We want this command to be un-interruptible so that is takes precedence over the gunner controls. Should look something like this: whileActiveOnce(new IntakeSequenceCommandGroup(m_intakeSub), false)

Reply:
fixed